### PR TITLE
Feature/scale

### PR DESCRIPTION
**Pull request by ArleneRangel**
### Changes Description
Fixed scale bug on clue_question_page + game_turn. made CSS changes to the scale. Added the team's score to the results_page. 

### Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### Changes Details
The scale bug wasn't allowing mirroring on multiple screens and disappearing as players submitted their clues. Improved the CSS scale to make it look less 'speedometer'. Added code to views to populate team's score in team_results page. 
List any dependencies that are required for this change_

### How Has This Been Tested?
Yes. We played the game in Safari + Chrome. 

### Checklist
- [x] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests.
- [ ] Do we need to implement analytics?
- [ ] Will this be part of a product update? If yes, please write one phrase about this update

### CODE REVIEW

File: app/templates/app/game_result.html [hunk 1]
```diff
@@ -15,16 +15,30 @@ <h1>Wavelength Tournament</h1>
...
     <div id='question_1'>
-    <img src='/static/img/question_clue_spectrum.png' alt="Left Right" />
-    <p style="text-align:left; width: 450px;">
-        {{question.left_spectrum}}
-        <span style="float:right;">
-            {{question.right_spectrum}}
-        </span>
-    </p>
-    <br>
+
```
Comment (AndresRoc) [inline app/templates/app/game_result.html:17]: Why the space?

File: app/static/css/scale.css [hunk 2]
```diff
@@ -24,4 +24,30 @@
   background-color: #F0C5C5;
 }
 
+#sliderTwo .rs-handle  {
```
Comment (AndresRoc) [inline app/static/css/scale.css:29]: Please add a comment to differentiate the code above with the code below

File: app/templates/app/game_turn.html [hunk 3]
```diff
@@ -10,7 +10,8 @@ <h2>YOUR CLUE</h2>
 
     <h1>{{clue.clue_given}}</h1>
    
-    <form action="{% url 'app:team_answer_response_form' game_id=game_id team_id=team_id player_id=player_id %}" method="post" onsubmit="return submitButtonRefresh()">
+
```
Comment (AndresRoc) [inline app/templates/app/game_turn.html]: Why the space?

File: app/templates/app/question_clue_spectrum.html [hunk 4]
```diff
@@ -103,6 +120,32 @@ <h1>Waiting for other teammates to submit their clues....</h1>
...
+            value: '{{generated_random_question_answer }}',
+            handleSize: 0,
+            handleShape: "square",
+            circleShape: "half-top",
+            min: 1,
+            max: 100,
+            disabled: true,
+    }); //end of .roundSlider 
+
+    $("#sliderTwo").roundSlider({
```
Comment (AndresRoc) [inline app/templates/app/question_clue_spectrum.html:135]: What is the difference between the code above and the one below? consider adding a comment to differentiate

File: app/urls.py [hunk 5]
```diff
@@ -25,12 +25,14 @@
     path('player_game_assignation/<int:game_id>', views.player_game_assignation, name='player_game_assignation'),
     path('player_game_assignation/<int:game_id>/<int:player_id>', views.player_game_assignation, name='player_game_assignation'),
     path("question_clue_spectrum/<int:game_id>/<int:team_id>/<int:player_id>", views.question_clue_spectrum, name='question_clue_spectrum'),
-    path("question_response_form", views.question_response_form, name="question_response_form"),
+    # path("question_response_form", views.question_response_form, name="question_response_form"),
```
Comment (AndresRoc) [inline app/urls.py]: Is this in use? if not lets delete it

File: app/urls.py [hunk 6]
```diff
@@ -25,12 +25,14 @@
...
     path("question_save", views.question_save, name="question_save"),
     path("scale", views.scale, name="scale"),
     path('team_creation/<int:game_id>/<int:player_id>', views.team_creation, name='team_creation'),
     path('team_page/<int:game_id>/<int:team_id>/<int:player_id>', views.team_page, name='team_page'),
     # path("team_score", views.team_score, name='team_score'),
```
Comment (AndresRoc) [inline app/urls.py]: Is this in use? if not lets delete it

File: app/urls.py [hunk 7]
```diff
@@ -25,12 +25,14 @@
...
     path('game_result/<int:game_id>/<int:team_id>/<int:player_id>/<int:turn_id>', views.game_result, name='game_result'),
-    path("waiting_room", views.waiting_room, name="waiting_room"),
+    # path('game_result/<int:turn_id>', views.game_result, name='game_result'),
```
Comment (AndresRoc) [inline app/urls.py]: Is this in use? if not lets delete it

File: app/views.py [hunk 8]
```diff
@@ -265,8 +265,15 @@ def question_clue_spectrum(request, game_id, team_id, player_id):
     # save the generated question into QuestionHistory
     question_history = QuestionHistory.objects.create(player=player, question=random_question)
     question_history2 = QuestionHistory.objects.create(player=player, question=random_question2)
+   
+    # save the generate answer into GameTurn
+    generated_random_question_answer = random.randint(1, 100)
+    team_answer = GameTurn.objects.filter(team=team, player=player).update(team_answer=generated_random_question_answer)
+    generated_random_question_answer_two = random.randint(1, 100)
+    print(str(generated_random_question_answer ) + " | this is question_answer")
+
```
Comment (AndresRoc) [inline app/views.py:273]: Why the extra space?

File: app/views.py [hunk 9]
```diff
@@ -311,23 +318,17 @@ def game_end(request, game_id):
 
 def game_turn(request, game_id, team_id, player_id):
     # game_turn spectrum has to be from team members
-    # check if spectrum already be used
```
Comment (AndresRoc) [inline app/views.py]: Extra space?

File: app/views.py [hunk 10]
```diff
@@ -336,7 +337,12 @@ def game_result(request, game_id, team_id, player_id, turn_id):
     team = Team.objects.get(id=team_id)
     turns_remaining = GameTurn.objects.filter(team=team, team_answer=0).count()
 
-    context = {"game_turn" : game_turn, "question" : question, "turns_remaining" : turns_remaining, "game_id" : game_id, "team_id" : team_id, "player_id" : player_id}
+    team_answer = game_turn.team_answer
+    question_answer = game_turn.question_answer
+
```
Comment (AndresRoc) [inline app/views.py:333]: too much extra space

File: app/templates/app/question_clue_spectrum.html [hunk 11]
```diff
@@ -45,9 +54,17 @@ <h1>Wavelength Tournament</h1>
```
Comment (AndresRoc) [inline app/templates/app/question_clue_spectrum.html:51]: Extra space

File: app/templates/app/question_clue_spectrum.html [hunk 12]
```diff
@@ -20,8 +20,17 @@ <h1>Wavelength Tournament</h1>
 <br>
     <hr>
```
Comment (AndresRoc) [inline app/templates/app/question_clue_spectrum.html:22]: Extra space

File: app/templates/app/question_clue_spectrum.html [hunk 13]
```diff
@@ -20,8 +20,17 @@ <h1>Wavelength Tournament</h1>
 <br>
     <hr>
 
-    <div id='question_1'>
-    <img src='/static/img/question_clue_spectrum.png' alt="Left Right" />
+<div id='question_1'>
```
Comment (AndresRoc) [inline app/templates/app/question_clue_spectrum.html]: Indentation doesn't seem to match

File: app/templates/app/question_clue_spectrum.html [hunk 14]
```diff
@@ -20,8 +20,17 @@ <h1>Wavelength Tournament</h1>
 <br>
     <hr>
```
Comment (AndresRoc) [inline app/templates/app/question_clue_spectrum.html]: Where is the closing tag for this?

File: app/templates/app/question_clue_spectrum.html [hunk 15]
```diff
@@ -45,9 +54,17 @@ <h1>Wavelength Tournament</h1>
 
     <hr>
```
Comment (AndresRoc) [inline app/templates/app/question_clue_spectrum.html]: Where is the closing tag for this?

**Review by AndresRoc — CHANGES_REQUESTED** [review]
Please resolve the comments mentioned in the review

File: app/templates/app/game_result.html (hunk 1 above)
Comment (ArleneRangel) [reply app/templates/app/game_result.html:17]: corrected

File: app/static/css/scale.css (hunk 2 above)
Comment (ArleneRangel) [reply app/static/css/scale.css:29]: added comments

File: app/templates/app/game_turn.html (hunk 3 above)
Comment (ArleneRangel) [reply app/templates/app/game_turn.html]: corrected

File: app/urls.py (hunk 5 above)
Comment (ArleneRangel) [reply app/urls.py]: deleted

File: app/urls.py (hunk 6 above)
Comment (ArleneRangel) [reply app/urls.py]: deleted

File: app/views.py (hunk 8 above)
Comment (ArleneRangel) [reply app/views.py:273]: deleted

File: app/views.py (hunk 9 above)
Comment (ArleneRangel) [reply app/views.py]: deleted

File: app/templates/app/question_clue_spectrum.html (hunk 11 above)
Comment (ArleneRangel) [reply app/templates/app/question_clue_spectrum.html:51]: resolved

File: app/templates/app/question_clue_spectrum.html (hunk 12 above)
Comment (ArleneRangel) [reply app/templates/app/question_clue_spectrum.html:22]: resolved

File: app/templates/app/question_clue_spectrum.html (hunk 13 above)
Comment (ArleneRangel) [reply app/templates/app/question_clue_spectrum.html]: fixed

File: app/templates/app/question_clue_spectrum.html (hunk 14 above)
Comment (ArleneRangel) [reply app/templates/app/question_clue_spectrum.html]: added

File: app/templates/app/question_clue_spectrum.html (hunk 15 above)
Comment (ArleneRangel) [reply app/templates/app/question_clue_spectrum.html]: added

File: app/urls.py (hunk 7 above)
Comment (ArleneRangel) [reply app/urls.py]: deleted

File: app/views.py (hunk 10 above)
Comment (ArleneRangel) [reply app/views.py:333]: no need

File: app/templates/app/question_clue_spectrum.html (hunk 4 above)
Comment (ArleneRangel) [reply app/templates/app/question_clue_spectrum.html:135]: added

**Review by AndresRoc — APPROVED** [review]
Approved

**Pull request merged**